assets/src/edit-story/elements/text/output.js

```diff
@@ -146,7 +134,7 @@ export function TextOutputWithUnits({
             <span
               style={backgroundTextStyle}
               dangerouslySetInnerHTML={{
-                __html: draftMarkupToContent(content, bold),
+                __html: getValidHTML(content),
```

Comment:
Ditto: if we are convinced that we save content correctly everywhere - no reason to validate it again here.

Reply:
Closing this one in favor of the other discussion.